nacc_attribute_deriver/attributes/mqt/scan.py

```diff
@@ -84,16 +91,19 @@ def _create_scan_pet_tau_tracers(self) -> Optional[str]:
 
         return None
 
-    def get_centiloid(self) -> Optional[float]:
-        """Get the centiloid value."""
-        centiloid = None
+    def get_pet_float(self, key: str, subprefix: PETPrefix):
+        """Get PET float value."""
         try:
-            self.__scan.assert_required(REQUIRED_FIELDS[PETPrefix.AMYLOID_PET_GAAIN])
-            centiloid = float(self.__scan.get_value("centiloids"))
+            self.__scan.assert_required(REQUIRED_FIELDS[subprefix])
+            return float(self.__scan.get_value(key))
         except (ValueError, TypeError):
-            return None
+            pass
 
-        return centiloid
+        return None
+
+    def get_centiloid(self) -> Optional[float]:
+        """Get the centiloid value."""
+        return self.get_pet_float("centiloids", PETPrefix.AMYLOID_PET_GAAIN)
 
     # Note: Be careful about the float return type here with the min computation.
```

Comment:
would you mind removing the note about the float type? the note applies to the min/max operation, and I believe I took care of it already